generated/src/aws-cpp-sdk-s3-crt/include/aws/s3-crt/S3CrtClientConfiguration.h

```diff
@@ -62,6 +65,80 @@ namespace Aws
             IdentityProviderSupplier identityProviderSupplier = [](const S3CrtClient &client) -> std::shared_ptr<S3ExpressIdentityProvider> {
                 return Aws::MakeShared<DefaultS3ExpressIdentityProvider>("S3CrtClientConfiguration", client);
             };
+
+            /* S3 CRT specifics */
+            S3CrtClientConfiguration(const S3CrtClientConfiguration&) = default;
+            S3CrtClientConfiguration& operator=(const S3CrtClientConfiguration&) = default;
+
+            struct S3CrtConfigFactories
+            {
+                /**
+                 * CRT retry strategy factory method.
+                 * Default will try to match SDK Retry Stratgey (which defaults to DefaultRetryStrategy)
+                 * Note: CRT retry strategy is not SDK retry strategy, retry algorithm and request counting are different.
+                 */
+                std::function<aws_retry_strategy*(const S3Crt::S3CrtClientConfiguration&)> retryStrategyCreateFn;
+
+                static S3CrtConfigFactories defaultFactories;
+            };
+
+            S3CrtConfigFactories crtConfigFactories = S3CrtConfigFactories::defaultFactories;
+
+            /** Client bootstrap used for common staples such as event loop group, host resolver, etc..
+             *  If this is nullptr, SDK will create a default one for you.
+             */
+            std::shared_ptr<Aws::Crt::Io::ClientBootstrap> clientBootstrap;
+
+            /** Size of parts the files will be downloaded or uploaded in. Useful for Put/GetObject APIs
+             *  defaults to 5MB, if user set it to be less than 5MB, SDK will set it to 5MB.
+             */
+            size_t partSize = 5 * 1024 * 1024;
+
+            /** TLS Options to be used for each connection.
+             *  If scheme is Https and tlsConnectionOptions is null, SDK will create a default one for you.
+             */
+            std::shared_ptr<Aws::Crt::Io::TlsConnectionOptions> tlsConnectionOptions;
+
+            /* Throughput target in Gbps that we are trying to reach. Normally it's the NIC's throughput */
+            double throughputTargetGbps = 2.0;
```

Comment:
i know its an old setting, but 10 is probably better. and those low values behave a bit weird due to rounding that currently happens in crt (i.e. there is no difference between any values you set below 4)